eZ/Publish/Core/MVC/Symfony/Matcher/AbstractMatcherFactory.php

```diff
@@ -1,154 +0,0 @@
-<?php
-
-/**
- * File containing the AbstractMatcherFactory class.
- *
- * @copyright Copyright (C) eZ Systems AS. All rights reserved.
- * @license For full copyright and license information view LICENSE file distributed with this source code.
- */
-namespace eZ\Publish\Core\MVC\Symfony\Matcher;
-
-use eZ\Publish\API\Repository\Repository;
-use eZ\Publish\Core\MVC\RepositoryAwareInterface;
-use eZ\Publish\Core\MVC\Symfony\View\View;
-use SplObjectStorage;
-use InvalidArgumentException;
-
-/**
- * Base for MatcherFactory classes.
- *
- * Implementors can define MATCHER_RELATIVE_NAMESPACE constant. If so, getMatcher() will return instances of objects relative
- * to this namespace if $matcherIdentifier argument doesn't begin with a '\' (FQ class name).
- *
- * @deprecated Deprecated since 6.0, will be removed in 6.1. Use ClassNameMatcherFactory instead.
- */
-abstract class AbstractMatcherFactory implements MatcherFactoryInterface
```

Comment:
Seems like this was yet another officially undocumented SPI-like extension point. Since this was documented here, this drop would need to be mentioned in `changes-8.0.md` as well.